runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -6692,63 +6832,66 @@ void TR_EscapeAnalysis::makeContiguousLocalAllocation(Candidate *candidate)
 
    if (candidate->isExplicitlyInitialized())
       {
-      // Find all the explicit zero-initializations and see if any of the
-      // generic int shadows can be replaced by real field references.
+      if (originalAllocationOpCode != TR::newvalue)
```

Comment:
I can do that.  There's also an `else` that's associated with the `if (candidate->isExplicitlyInitialized())`, so I'll have to change the entire thing from this form

```
if (candidate->isExplicitlyInitialized())
   {
   if (originalAllocationOpCode != TR::newvalue)
   ...
   }
else
   ...
```

to this form

```
if (candidate->isExplicitlyInitialized() && (originalAllocationOpCode != TR::newvalue))
   ...
   }
else if (!candidate->isExplicitlyInitialized())
   ...
```

Reply:
Now I see.  Thanks for pointing it out. On a second thought, it's probably better to keep the change as it is so that it's easier to follow the logic. When I was reviewing this part of the code, I was more thinking of how easy it is to compare the change to the previous version or existing master branch. However, now I think what's more important is how easy it is to follow the logic with this change.